src/EditorFeatures/Core.Cocoa/Preview/PreviewPaneService.cs

```diff
@@ -101,7 +101,6 @@ private NSImage GetSeverityIconForDiagnostic(DiagnosticData diagnostic)
             if (data.Properties.TryGetValue("OptionName", out _))
```

Comment:
Delete the entire if here since it's not doing anything?

Reply:
Done.